### PR TITLE
Commerce 7649 - Add or Update of product relations fix

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-7649

With this PR will be possible to add or update product relation via REST APIs using just the product involved ids.

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
@Commando18 can you have a look at this?

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkLocalServiceImpl.java [hunk 1]
```diff
@@ -203,6 +203,14 @@ public void deleteCPDefinitionLinksByCProductId(long cProductId)
 		}
 	}
 
+	@Override
+	public CPDefinitionLink fetchByC_C_T(
```
Comment (Commando18) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkLocalServiceImpl.java:207]: rename method to fetchCPDefinitionLink

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkServiceImpl.java [hunk 2]
```diff
@@ -68,6 +68,17 @@ public void deleteCPDefinitionLink(long cpDefinitionLinkId)
 		cpDefinitionLinkLocalService.deleteCPDefinitionLink(cpDefinitionLinkId);
 	}
 
+	@Override
+	public CPDefinitionLink fetchByC_C_T(
```
Comment (Commando18) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkServiceImpl.java:72]: rename method to fetchCPDefinitionLink

**Review by Commando18 — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**